canonical-web-and-design/snapcraft.io
# Vanilla snap details page

Fixes https://github.com/canonical-web-and-design/snapcraft.io/issues/2130

## QA
- Pull the branch
- `./run`
- Visit multiple snaps http://0.0.0.0:8004/toto http://0.0.0.0:8004/spotify http://0.0.0.0:8004/slack http://0.0.0.0:8004/vlc etc.
- Ensure everything works and looks as expected specifically:
  - channel map
  - video/ image layout
  - distro install instruction
  - all the above at various window sizes (mobile, tablet etc.)
- Visit multiple snap distro install pages http://0.0.0.0:8004/install/toto/arch http://0.0.0.0:8004/install/spotify/debian etc.
- Ensure everything looks and works as expected specifically:
  - "Install" button jump
  - "Show more" link on description
